tests/sklearn/test_sklearn_models.py

```diff
@@ -1153,35 +1146,36 @@ def test_correctness_with_sklearn(
     )
 
 
-@pytest.mark.parametrize("model_class, parameters", MODELS_AND_DATASETS)
+# Neural network hyper-parameters are not tested
 @pytest.mark.parametrize(
-    "n_bits",
-    N_BITS_WEEKLY_ONLY_BUILDS + N_BITS_REGULAR_BUILDS,
+    "model_class, parameters",
+    get_sklearn_linear_models_and_datasets()
+    + get_sklearn_tree_models_and_datasets()
+    + get_sklearn_neighbors_models_and_datasets(),
 )
 def test_hyper_parameters(
     model_class,
     parameters,
-    n_bits,
     load_data,
     check_r2_score,
     check_accuracy,
     is_weekly_option,
     verbose=True,
 ):
     """Testing hyper parameters."""
+
+    n_bits = N_BITS_THRESHOLD_FOR_SKLEARN_CORRECTNESS_TESTS
```

Comment:
the whole point of this test is to load hyper-parameters and test correctness, so let's use this default parameter here as well